PYME/LMVis/pipeline.py

```diff
@@ -867,6 +858,8 @@ def OpenChannel(self, filename='', ds=None, channel_name='', **kwargs):
         if ds is None:
             #load from file
             ds = self._ds_from_file(filename, **kwargs)
+            self.mdh.copyEntriesFrom(ds.mdh)
```

Comment:
`OpenChannel` is specifically for adding / opening an additional channel (after the main data is loaded). It should **not** overwrite the pipeline level metadata and/or events.